src/components/AddMovie/AddMovie.css

```diff
@@ -0,0 +1,16 @@
+.button--transparent {
+	background: rgba(96, 96, 96, 0.68);
+	border-color: transparent;
+	border-radius: 4px;
+	width: 177px;
+	height: 46px;
+	font-family: 'Montserrat', sans-serif;
+	font-style: normal;
+	font-weight: 600;
+	font-size: 20px;
```

Comment:
It is also better to use rem where ever you can because it respects browser user preferences

Reply:
Ack - would you recommend to do not focus on requirements from figma project and for example change px for rem?